npm/pkg/dataplane/dataplane_windows_test.go

```diff
@@ -99,19 +99,19 @@ func TestRemoveCommonEndpoints(t *testing.T) {
 			name:              "1 value same",
 			endpoints:         []hcn.HostComputeEndpoint{{Id: "456901"}, {Id: "123456"}, {Id: "560971"}},
 			endpointsAttached: []hcn.HostComputeEndpoint{{Id: "567890"}, {Id: "123456"}, {Id: "789012"}},
-			expected:          []hcn.HostComputeEndpoint{{Id: "567890"}, {Id: "123456"}, {Id: "789012"}, {Id: "456901"}, {Id: "560971"}},
+			expected:          []hcn.HostComputeEndpoint{{Id: "456901"}, {Id: "123456"}, {Id: "560971"}, {Id: "567890"}, {Id: "789012"}},
```

Comment:
just an FYI in case you need in the future. Sometimes it's easiest to sort both the output and the expected output for unit tests. That's a bit simpler for slices of strings versus structs but

Reply:
got it makes sense, thank you this is good to know for future use